lib/tasks/deployment/20230725161734_remove_meter_consolidation_alert.rake

```diff
@@ -0,0 +1,23 @@
+namespace :after_party do
+  desc 'Deployment task: remove_meter_consolidation_alert'
+  task remove_meter_consolidation_alert: :environment do
+    puts "Running deploy task 'remove_meter_consolidation_alert'"
+
+    #Delete unneeded attributes
+    GlobalMeterAttribute.where(attribute_type: "indicative_standing_charge").destroy_all
+
```

Comment:
is it worth wrapping this in a transaction?

Reply:
We don't often do that for these changes. I don't think there will be any issues (I've run locally to test).